scipy/optimize/tests/test_linprog.py

```diff
@@ -1367,51 +1359,82 @@ def test_network_flow(self):
 
 
 class TestLinprogSimplexDefault(LinprogSimplexTests):
-    options = {}
 
-    def test_bug_7237(self):
-        # we want to check for the error
+    def setup_method(self):
+        self.options = {}
+
+    def test_bug_5400(self):
+        with pytest.raises(ValueError):
+            super(TestLinprogSimplexDefault, self).test_bug_5400()
```

Comment:
I've read the responses about `ValueError`; I'm still not convinced this is an appropriate use of the error (even if I once was). But this is not new to this PR, so let's let it be.